doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -0,0 +1,113 @@
+
+	This guide assumes that you are starting without an existing Python 3 installation. If you already have Python 3, you might want to uninstall it first to avoid ambiguity over which Python version is being used at the command line.
+
+.. _quickstart-ubuntu-build:
+
+Building SciPy
+--------------
+
+*Consider watching the companion video* `Anaconda SciPy Dev: Part I (macOS)`_ *before starting. Even though the video is intended for macOS users, most of the procedures are the same or very similar.*
+
+#. Download, install, and test the latest release of the `Anaconda Distribution of Python`_. In addition to the latest version of Python 3, the Anaconda distribution includes dozens of the most popular Python packages for scientific computing, the Spyder integrated development environment (IDE), the ``conda`` package manager, and tools for managing virtual environments.
+
+   In Ubuntu, the Anaconda download comes as a ``.sh`` file. `In a terminal, navigate <https://help.ubuntu.com/community/UsingTheTerminal>`_ to the directory in which the ``.sh`` file is located and enter ``./file.sh``, where ``file.sh`` is to be replaced with the full name of the Anaconda download. This starts the installation process; from there, simply follow the prompts.
+
+#. Install git. An easy way to do this is to enter the command ``sudo apt-get install git`` in the terminal and follow the prompts. We'll use this software to download and manage the SciPy source code.
+
+#. Browse to the `SciPy repository on GitHub <https://github.com/scipy/scipy>`_ and `create your own fork <https://help.github.com/en/articles/fork-a-repo>`_. You'll need to create a GitHub account if you don't already have one.
+
+#. Browse to your fork. Your fork will have a URL like `https://github.com/mdhaber/scipy <https://github.com/mdhaber/scipy>`_, except with your GitHub username in place of "mdhaber".
+
+#. Click the big, green "Clone or download" button, and copy the ".git" URL to the clipboard. The URL will be the same as your fork's URL, except it will end in ".git".
+
+#. Create a folder for the SciPy source code in a convenient place on your computer. Navigate to it in the terminal.
+
+#. Enter the command ``git clone`` followed by your fork's .git URL. Note that this creates in the terminal's working directory a ``scipy`` folder containing the SciPy source code.
+
+#. In the terminal, navigate into the ``scipy`` root directory (e.g. ``cd scipy``).
+
+#. Install `Homebrew on Linux`_. Enter into the terminal |br| ``sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"`` |br| or follow the installation instructions listed on the `Homebrew on Linux`_ website. Homebrew requires additional packages to be installed, and lists the required commands in the terminal window. Copy and paste these commands into the terminal to complete the Homebrew setup. If no additional commands are given, they may be found `here <https://docs.brew.sh/Homebrew-on-Linux>`_.
```

Comment:
Using Homebew on Linux doesn't really make sense to me, is there a reason we require *"Homebew on Linux"* to install gcc?  Were already installing git via apt. Installing gcc with apt is simply: ``sudo apt install gcc``.

Reply:
@fletcheaston tried that and that version of `gcc` seemed not to work for him. We tried the Homebrew version and it did work.

Reply:
Ok. From memory I needed to install g++ individually.